static/js/map/map.settings.js

```diff
@@ -27,6 +27,8 @@ function initSettings() {
     }
     if (serverSettings.pokemonValues) {
         settings.filterPokemonByValues = Store.get('filterPokemonByValues')
+        settings.filterPokemonBySize = Store.get('filterPokemonBySize')
+        settings.filterPokemonBySize = Store.get('filterPokemonBySizeOptions')
```

Comment:
Here is the problem why filters are not applied on reload. 

settings.filterPokemonBySizeOptions = Store.get('filterPokemonBySizeOptions')

Reply:
Nice catch. Have made these changes, but can't actually test it at the moment.